src/js/engagement_view/Makefile

```diff
@@ -108,7 +108,7 @@ shell: ## Drop into a shell inside the build container
 
 .PHONY: start
 start: extra_docker_run_args := --interactive --tty --network host
-start: build-image
+start: build-code
```

Comment:
This suggests that you need to `yarn build` (make a release build) before you `yarn start` (which is the debugging one that auto-updates I believe), which seems incorrect.

Reply:
Leaving per cm